src/lib/components/metrics-definition/DimensionColumns.ts

```diff
@@ -1,24 +1,74 @@
-import EditableTableCell from "$lib/components/table/EditableTableCell.svelte";
+import EditableTableCell from "$lib/components/table-editable/EditableTableCell.svelte";
 import type { DimensionDefinitionEntity } from "$common/data-modeler-state-service/entity-state-service/DimensionDefinitionStateService";
-import type { ColumnConfig } from "$lib/components/table/ColumnConfig";
-import RowActionsCell from "$lib/components/table/RowActionsCell.svelte";
+import type { ColumnConfig } from "$lib/components/table-editable/ColumnConfigumnConfig";
+import RowActionsCell from "$lib/components/table-editable/RowActionsCell.svelte";
 
-export const DimensionColumns: Array<ColumnConfig> = [
-  "sqlName",
-  "dimensionColumn",
-  "description",
-].map((col) => ({
-  name: col,
-  type: "VARCHAR",
-  renderer: EditableTableCell,
-}));
-DimensionColumns[0].validation = (row: DimensionDefinitionEntity) =>
-  row.sqlNameIsValid;
-DimensionColumns[1].validation = (row: DimensionDefinitionEntity) =>
-  row.dimensionIsValid;
-DimensionColumns.push({
-  name: "id",
-  label: "Actions",
-  type: "VARCHAR",
-  renderer: RowActionsCell,
-});
+// export const DimensionColumns: Array<ColumnConfig> = [
+//   "sqlName",
+//   "dimensionColumn",
+//   "description",
+// ].map((col) => ({
+//   name: col,
+//   type: "VARCHAR",
+//   renderer: EditableTableCell,
+// }));
+// DimensionColumns[0].validation = (row: DimensionDefinitionEntity) =>
+//   row.sqlNameIsValid;
+// DimensionColumns[1].validation = (row: DimensionDefinitionEntity) =>
+//   row.dimensionIsValid;
+// DimensionColumns.push({
+//   name: "id",
+//   label: "Actions",
+//   type: "VARCHAR",
+//   renderer: RowActionsCell,
+// });
+
+export const DimensionColumns: ColumnConfig[] = [
+  {
+    name: "labelSingle",
+    label: "label (single)",
+    tooltip: "a human readable name for this dimension",
+    renderer: EditableTableCell,
+  },
+
+  {
+    name: "dimensionColumn",
+    label: "dimension column",
+    tooltip:
+      "a categorical column from the data model that this metrics set is based on",
+    renderer: EditableTableCell,
```

Comment:
Validator is missing here based on `dimensionIsValid`

Reply:
thank you for catching this!